packages/ses/scripts/bundle.js

```diff
@@ -15,40 +17,59 @@ const write = async (target, content) => {
   await fs.promises.writeFile(location, content);
 };
 
-const main = async () => {
+/**
+ * @param {object} [options]
+ * @param {string} [options.buildType] Suffix used to build special bundles (e.g. 'hermes')
+ */
+const writeBundle = async ({ buildType } = {}) => {
   const text = await fs.promises.readFile(
     fileURLToPath(`${root}/package.json`),
     'utf8',
   );
   const packageJson = JSON.parse(text);
   const version = packageJson.version;
 
+  const moduleTransforms = {};
+
+  let bundleFilePaths = [
+    'dist/ses.cjs',
+    'dist/ses.mjs',
+    'dist/ses.umd.js',
+    'dist/lockdown.cjs',
+    'dist/lockdown.mjs',
+    'dist/lockdown.umd.js',
+  ];
+  let terseFilePaths = ['dist/ses.umd.min.js', 'dist/lockdown.umd.min.js'];
+
+  if (buildType === 'hermes') {
+    bundleFilePaths = ['dist/ses-hermes.cjs'];
+    terseFilePaths = [];
+    Object.assign(moduleTransforms, hermesTransforms);
+  }
+
```

Comment:
```suggestion
  const { moduleTransforms, bundleFilePaths, terseFilePaths } = buildType === 'hermes'
  ? {
      moduleTransforms: hermesTransforms,
      bundleFilePaths: ['dist/ses-hermes.cjs'],
      terseFilePaths: [],
    } : {
      moduleTransforms: {},
      bundleFilePaths: [
        'dist/ses.cjs',
        'dist/ses.mjs',
        'dist/ses.umd.js',
        'dist/lockdown.cjs',
        'dist/lockdown.mjs',
        'dist/lockdown.umd.js',
      ],
      terseFilePaths: [
        'dist/ses.umd.min.js',
        'dist/lockdown.umd.min.js'
      ],
    };

Reply:
perhaps worth keeping the `if` statement, to potentially support transforming/bundling more `buildType`s in future?

Reply:
those other `buildType`s may be one or more of
- https://github.com/tc39/eshost?tab=readme-ov-file#eshostcreateagenttype-string-options---agent
- https://github.com/GoogleChromeLabs/jsvu?tab=readme-ov-file#supported-engines-per-os

Reply:
If more mutually exclusive `buildType`s come along a `switch`/`case` might be a good fit?